base/_buttons.scss

```diff
@@ -0,0 +1,63 @@
+@charset "UTF-8";
+
+// buttons.scss
+// This file contains all styles related to the button element.
+// see http://bitters.bourbon.io
+// dependent on utilities/color.scss
```

Comment:
I'd be willing to test it out. It seems like it will definitely help and make our package more robust. 👍